src/transactions/typeTag/typeTag.ts

```diff
@@ -0,0 +1,380 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
+import { Deserializer } from "../../bcs/deserializer";
+import { Serializable, Serializer } from "../../bcs/serializer";
+import { AccountAddress } from "../../core";
+import { Identifier } from "../instances/identifier";
+import { TypeTagVariants } from "../../types";
+
+export abstract class TypeTag extends Serializable {
```

Comment:
I dont think this file is part of this PR? you might need to rebase?

Reply:
so I moved `typeTag/index.ts` to `typeTag/typeTag.ts` and then export the other two files in there from `index.ts`

It looks like it's a new file/out of place, but it's just because I moved the file from `index.ts` to `typeTag.ts`

Reply:
gotcha. typeTag/typeTag.ts it a bit weird. can we rename?